src/elements/emby-ratingbutton/emby-ratingbutton.js

```diff
@@ -57,12 +57,12 @@ define(['connectionManager', 'serverNotifications', 'events', 'globalize', 'emby
 
     function setState(button, likes, isFavorite, updateAttribute) {
 
-        var icon = button.querySelector('i');
+        var icon = button.querySelector('.material-icons');
 
         if (isFavorite) {
 
             if (icon) {
-                icon.innerHTML = 'favorite';
+                icon.classList.add('favorite');
```

Comment:
why you do `.classList.add()` here but `.replace()` in other places?

Reply:
Nothing to replace - 'favorite' is everywhere.

Ideally, `remove` should be at the beginning. Like here
https://github.com/jellyfin/jellyfin-web/blob/d98645135b53612d852bf14991d247880f502265/src/components/nowplayingbar/nowplayingbar.js#L331-L344